pkg/tsdb/grafana-pyroscope-datasource/instance.go

```diff
@@ -6,6 +6,7 @@ import (
 	"fmt"
 	"net/http"
 	"net/url"
+	"strconv"
```

Comment:
Can we remove this?

Reply:
This is used to parse the start/end from the query params into ints. 

https://github.com/grafana/grafana/blob/8e01744e32e0c32ff3a2f68b8d526d307a22f43a/pkg/tsdb/grafana-pyroscope-datasource/instance.go#L97-L110